### PR TITLE
Added matomo events for home page, type of business and other type of…

### PR DESCRIPTION
**Pull request by ttingle-ch**
Added matomo events for home page, type of business and other type of business pages. Refactored common views

### CODE REVIEW

**Review by hemerem-ch — APPROVED** [review]

File: src/main/views/features/sole-trader/sector-you-work-in/sector-you-work-in.njk [hunk 1]
```diff
@@ -1,73 +0,0 @@
-{% from "govuk/components/radios/macro.njk"              import govukRadios %}
```
Comment (swalke-ch) [inline src/main/views/features/sole-trader/sector-you-work-in/sector-you-work-in.njk:1]: Why is this file deleted?

Comment (ttingle-ch) [reply src/main/views/features/sole-trader/sector-you-work-in/sector-you-work-in.njk:1]: It was a duplicate file

**Review by swalke-ch — APPROVED** [review]
